src/framework/shortcuts/view/editshortcutmodel.cpp

```diff
@@ -110,6 +110,71 @@ void EditShortcutModel::inputKey(Qt::Key key, Qt::KeyboardModifiers modifiers)
     emit newSequenceChanged();
 }
 
+bool EditShortcutModel::isShiftAllowed(Qt::Key key, Qt::KeyboardModifiers modifiers)
+{
+    if (!(modifiers & Qt::ShiftModifier)) {
+        return true;
+    }
```

Comment:
This particular condition shouldn't be part of the function. Sorry if that wasn't clear from my instructions.

The function needs to behave as its name implies. It should answer the question "Is Shift allowed?". The answer to that question shouldn't depend on whether the Shift key is currently being pressed.

Reply:
Sorry for that, Updated now